service-python/pdfgenerator/examples/sample_hypertension.json

```diff
@@ -0,0 +1,98 @@
+{
```

Comment:
Since this file is not referenced/used in the code, it may be accidentally deleted. Either add comments for how to use this file, reference it in a test, or use it as the default `input` value for the `__main__` block.